### PR TITLE
Ports Sofas from Paradise

### PR DESCRIPTION
**Pull request by Brasnak**
Ports Paradise sofas with minor improvements. Expect me to improve upon them later on.

:cl: optional name here
add: sofas
add: metal sheet recipe for sofas
imageadd: icons for sofas in chairs.dmi
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -21,6 +21,9 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 	new/datum/stack_recipe("swivel chair", /obj/structure/chair/office/dark, 5, one_per_turf = TRUE, on_floor = TRUE), \
 	new/datum/stack_recipe("comfy chair", /obj/structure/chair/comfy/beige, 2, one_per_turf = TRUE, on_floor = TRUE), \
 	new/datum/stack_recipe("bed", /obj/structure/bed, 2, one_per_turf = TRUE, on_floor = TRUE), \
+	new/datum/stack_recipe("sofa (middle)", /obj/structure/chair/sofa, one_per_turf = TRUE, on_floor = TRUE), \
+	new /datum/stack_recipe("sofa (left)", /obj/structure/chair/sofa/left, one_per_turf = TRUE, on_floor = TRUE), \
```
Comment (SamHPurp) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: keep your spacing consistent please

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -21,6 +21,9 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
...
+	new /datum/stack_recipe("sofa (right)", /obj/structure/chair/sofa/right, one_per_turf = TRUE, on_floor = TRUE), \
```
Comment (SamHPurp) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: keep your spacing consistent please

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 3]
```diff
@@ -331,3 +331,18 @@
 	desc = "You sit in this. Either by will or force. Looks REALLY uncomfortable."
 	icon_state = "chairold"
 	item_chair = null
+
+/obj/structure/chair/sofa
+	name = "old ratty sofa"
+	icon_state = "sofaconnectors"
+	item_chair = null
+	anchored = 1
```
Comment (SamHPurp) [inline code/game/objects/structures/beds_chairs/chair.dm]: `TRUE`

Comment (Brasnak) [reply code/game/objects/structures/beds_chairs/chair.dm]: things that happen if you just copy paste...

**Comment by taukausanake** [comment]
It would be nice if sofas could smooth together instead of needing to be rotated in place. Also if they wouldn't rotate at all but that's because chairs

**Pull request closed without merging**